x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/25_semantic_text_field_mapping_chunking_bwc.yml

```diff
@@ -99,6 +99,20 @@ setup:
                   max_chunk_size: 10
                   overlap: 1
 
+  - do:
+      indices.create:
```

Comment:
I think we need to set legacy format here

Reply:
Thanks, https://github.com/elastic/elasticsearch/pull/129150/commits/f9d4d47da7c371dfff57850e918ceab7d6fa2108